javascript/app.js

```diff
@@ -22,3 +23,194 @@ function closeMenuHandler() {
+    technologies: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstraps'],
+    urlMobile: 'images/work_3.jpg',
+    urlDesktop: 'images/work_3_lg.png',
+    urlModal: '/images/work_modal.png',
+    alt: 'work_tonic_daily_personalized_reads',
+    live: 'https://dennis-every.github.io/',
+    source: 'https://github.com/dennis-every/portfolio',
+  },
+  {
+    id: 3,
+    title: 'Multi-Post Stories',
+    experiences: ['Canopy', 'Back End Dev', '2015'],
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    technologies: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstraps'],
+    urlMobile: 'images/work_4.jpg',
+    urlDesktop: 'images/work_4_lg.png',
+    urlModal: '/images/work_modal.png',
+    alt: 'work_multi-post_stories_daily_personalized_reads',
+    live: 'https://dennis-every.github.io/',
+    source: 'https://github.com/dennis-every/portfolio',
+  },
+];
+
+const worksLoadHandler = () => {
+  const works = document.getElementById('works');
+  worksArray.forEach((work) => {
+    const workCard = document.createElement('li');
+    workCard.classList.add('work-card');
+    workCard.innerHTML = `
+      <article class='work-card__article'>
+        <div class='work-card__header'>
+          <picture>
+            <source srcset=${work.urlDesktop} media='(min-width: 768px)' />
+            <img
+              src=${work.urlMobile}
+              alt=${work.alt}
+            />
+          </picture>
+        </div>
+        <div class='work-card__body'>
+          <div class='elements'>
+            <header class='work-card__title'>${work.title}</header>
+            <div>
+              <ul class='work-card__experience'>
+                <li>${work.experiences[0]}</li>
+                <li>&#x2022;</li>
+                <li>${work.experiences[1]}</li>
+                <li>&#x2022;</li>
+                <li>${work.experiences[2]}</li>
+              </ul>
+            </div>
+            <p class='work-card__description'>${work.description}</p>
+            <div>
+              <ul class='work-card__technologies'>
+                <li>${work.technologies[0]}</li>
+                <li>${work.technologies[1]}</li>
+                <li>${work.technologies[2]}</li>
+              </ul>
+            </div>
+            <div class='btn-container'>
+              <button type='button' class='btn'>
+                <a href=${work.live}>See Project</a>
+              </button>
+            </div>
```

Comment:
- [ ] Great job so far 👏🏻 However all the `See Project` doesn't work as shown on the video screen 👇🏻 Actually they are working as an internal link to the home section as shown on the screenshot video. When I click on one, of them the URL link change to this 👉🏻   `https://dennis-every.github.io/#`/. Kindly let's fix this and make sure that: 
-  When the user clicks the button to check project details, the popup with details about the project appears.✔️   
- When the user clicks the close button, the popup disappears ✔️ 



[Screencast from 12-21-2022 05:37:39 AM.webm](https://user-images.githubusercontent.com/79058364/208816682-79f0ce76-678c-401b-8c6d-7e7e9a8665f5.webm)


- [ ] Please make sure The popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, text, margins).